content/guides/dogstatsd.md

```diff
@@ -298,7 +298,7 @@ the packets:
 #### Fields
 - Mandatory:
   - `title` — Event title.
-  - `text` — Event text. Supports line breaks.
+  - `text` — Event text. Insert line breaks with an escaped slash (`\\n`)
```

Comment:
@technovangelist hmm, yes, it seems we're documenting in three different places what the two required fields are for events. This is totally unnecessary, I'll remove these soon in a followup to this PR.